modules/gdscript/tests/gdscript_test_runner_suite.h

```diff
@@ -48,8 +48,10 @@ TEST_SUITE("[Modules][GDScript]") {
 		REQUIRE_MESSAGE(fail_count == 0, "All GDScript tests should pass.");
 	}
 }
+#endif // TOOLS_ENABLED
 
 TEST_CASE("[Modules][GDScript] Load source code dynamically and run it") {
+	GDScriptLanguage::get_singleton()->init();
```

Comment:
Does this solve the TODO on line 39?

Reply:
No, the `Script compilation and runtime` test still fails, and it only fails in release builds unlike these ones which fail when alone. 
Unfortunately it is more complex to fix.